packages/shylu/fastilu/src/shylu_fastic.hpp

```diff
@@ -194,8 +194,8 @@ class FastICPrec
             Kokkos::parallel_for(nnzL, copyFunc3);
             double t = timer.seconds();
 
-            std::cout << "Symbolic phase complete." << std::endl;
-            std::cout << "Init time: "<< t << "s" << std::endl;
+            //std::cout << "Symbolic phase complete." << std::endl;
```

Comment:
I'm guarding them in ifdef FASTIC_DEBUG_OUTPUT etc. I'm not using ifdef SHYLU_DEBUG because I imagine there will be users who need a debug build but don't need this output.